src/main/resources/db/changelog/changes/04-create-users-roles-table.yaml

```diff
@@ -0,0 +1,22 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users_roles-table
+      author: root
+      changes:
+        - createTable:
+            tableName: users_roles
+            columns:
```

Comment:
Declare also a primary key constraint on both columns (so each entry of user_id + role_id is unique)